core/src/collections.rs

```diff
@@ -341,45 +342,45 @@ mod unit {
 
     #[test]
     fn metadata_column_nullability() {
-        use crate::col_metadata::ColumnNullability;
         use crate::{collections::MongoCollections, stmt::MongoStatement};
+        use odbc_sys::Nullability;
         assert_eq!(
-            ColumnNullability::NoNulls,
+            Nullability::NO_NULLS,
             MongoCollections::empty()
                 .get_col_metadata(1)
                 .unwrap()
-                .is_nullable
+                .nullability
         );
         assert_eq!(
-            ColumnNullability::Nullable,
+            Nullability::NULLABLE,
             MongoCollections::empty()
                 .get_col_metadata(2)
                 .unwrap()
-                .is_nullable
+                .nullability
         );
-        // Docs do not say NoNulls, but there is no way the tale name can be null.
+        // Docs do not say NO_NULLS, but there is no way the tale name can be null.
         assert_eq!(
-            ColumnNullability::NoNulls,
+            Nullability::NO_NULLS,
             MongoCollections::empty()
                 .get_col_metadata(3)
                 .unwrap()
-                .is_nullable
+                .nullability
         );
-        // The docs also do not say NoNulls, but they enumerate every possible value and
+        // The docs also do not say NO_NULLS, but they enumerate every possible value and
         // NULL is not one of them.
         assert_eq!(
-            ColumnNullability::NoNulls,
+            Nullability::NO_NULLS,
             MongoCollections::empty()
                 .get_col_metadata(4)
                 .unwrap()
-                .is_nullable
+                .nullability
         );
         assert_eq!(
-            ColumnNullability::Nullable,
+            Nullability::NULLABLE,
             MongoCollections::empty()
                 .get_col_metadata(5)
                 .unwrap()
-                .is_nullable
```

Comment:
`is_nullable` really portrays this value as a boolean, I changed it to be more clear.